apps/dashboard/src/components/file-upload.tsx

```
@@ -0,0 +1,83 @@
+    );
+
+    return (
+        <Card>
+            <CardHeader>
+                <CardTitle>Upload Documents</CardTitle>
+                <p className="text-sm text-muted-foreground mt-1">
+                    Upload documents to process with this workflow. Supports PDF and image files.
+                </p>
+            </CardHeader>
+            <CardContent className="p-0">
+                {/** biome-ignore lint/a11y/noStaticElementInteractions: drag and drop functionality requires these interactions */}
```

Biome (1.9.4)

[error] failed to parse category "lint/a11y/noStaticElementInteractions"



(suppressions/parse)